yajsapi/executor/ctx.ts

```diff
@@ -233,11 +238,9 @@ class _RecvFile extends Work {
 
   async post(): Promise<void> {
     if (!this._dst_slot) throw "_RecvFile post without prepare";
-    if (this._emitter)
-      this._emitter(new events.DownloadStarted({ path: this._src_path }));
+    if (this._emitter) this._emitter(new events.DownloadStarted({ path: this._src_path }));
```

Comment:
wouldn't this be more readable: 
```suggestion
    if (this._emitter) { 
        this._emitter(new events.DownloadStarted({ path: this._src_path }));
    }
```
?

Reply:
Yes, i agree